src/mumble/AudioOutput.h

```diff
@@ -73,9 +73,10 @@ class AudioOutput : public QThread {
 		Q_OBJECT
 		Q_DISABLE_COPY(AudioOutput)
 	private:
-		float *fSpeakers;
+		float *fSpeakers; // Speaker positional vector
```

Comment:
```suggestion
		/// Speaker positional vector
		float *fSpeakers;
```
-> conversion to proper Doxygen style comment